IPython/html/static/edit/js/main.js

```diff
@@ -75,4 +77,14 @@ require([
         }
     };
 
+    // Make sure the codemirror editor is sized appropriatley.
+    var _handle_resize = function() {
+        var header = $('#header');
+        var padding = header.outerHeight(true) - header.innerHeight();
+        $('div.CodeMirror').height(window.innerHeight - header.height() - 2*padding);
+    };
+    window.onresize = _handle_resize;
+
+    // On document ready, resize codemirror.
+    $(_handle_resize);
```

Comment:
Yeah, admittedly the first time I saw the JQuery constructor used this way I was confused too.  I'm probably just passing that confusion along by using it like that again, but it's ~15 or so less characters to type ;)